build.gradle

```
@@ -99,6 +104,11 @@ subprojects {
     project.licenseFile = project.rootProject.file('licenses/APACHE-LICENSE-2.0.txt')
     project.noticeFile = project.rootProject.file('NOTICE.txt')
   }
+
+  // Although the plugin is applied at the top-level, that doesn't make
+  // it available on subprojects.
+  project.apply plugin: 'com.github.sherter.google-java-format'
+  check.dependsOn 'verifyGoogleJavaFormat'
```

`precommit.check` should have worked  in the dame way, I do think we need that there.

---

```suggestion
  precommit.check.dependsOn 'verifyGoogleJavaFormat'
```

I tried the above but Gradle didn't like it. I'll keep looking it at, all suggestions welcome.